script/Registry.s.sol

```diff
@@ -58,49 +51,37 @@ uint256 constant MIN_DELAY = 3 days; // timelock for governor
 uint256 constant ORACLE_INTERVAL_PROD = 1 hours;
 
 // Vanity address params - use `cast create2` to find salt
-uint256 constant MAINNET_SALT_SYSTEMCOIN = 0;
-uint256 constant MAINNET_SALT_PROTOCOLTOKEN = 0;
-uint256 constant MAINNET_SALT_VAULT721 = 0;
+uint256 constant MAINNET_SALT_SYSTEMCOIN = 1;
+uint256 constant MAINNET_SALT_PROTOCOLTOKEN = 1;
+uint256 constant MAINNET_SALT_VAULT721 = 1;
 address constant MAINNET_CREATE2_FACTORY = address(0);
 
+// Camelot Relayer (pre-deployed @ sol 0.7.6)
+address constant MAINNET_CAMELOT_RELAYER_FACTORY = address(0);
+address constant MAINNET_CAMELOT_RELAYER = address(0);
+
+// Camelot Protocol
+address constant CAMELOT_AMM_FACTORY = 0x1a3c9B1d2F0529D97f2afC5136Cc23e58f1FD35B;
```

Comment:
Is the naming convention here different from testnet? I don't see this one above